app/src/store/data.ts

```diff
@@ -6,7 +6,7 @@
 import { computed, markRaw, Ref, ref } from 'vue';
 
 // --- Our imports ---
-import { BigNumber, Contract } from 'src/utils/ethers';
+import { BigNumber, Contract, formatUnits } from 'src/utils/ethers';
```

Comment:
This is a big file, and I'm no longer certain exactly what's happening in this file, what order data is being fetched, which things are only fetched once, etc. Any suggestions on how to make this more clear? Just comments? Or maybe can this be cleaned up by extracting helper methods? We don't have to do it in this PR, just wanted to bring attention to it

Reply:
All fetching methods have been moved into a new directory (`./app/src/utils/data/*`) - The dataStore should be more manageable now 👍